examples/experimental/Length of Stay demo on MIMIC/example/test_trainer.py

```diff
@@ -0,0 +1,100 @@
+"""module containing functions showing how to use the other functions in the
+    Length of Stay demo on MIMIC folder
+"""
+import os
+import sys
+import json
+import numpy as np
+import logging
+import torch.nn as nn
+from pprint import pformat
+from torch import randint as torch_randint
+from torch import float as torch_float
+from torch.utils.data import TensorDataset
+from torch.utils.data import DataLoader
+from torch.utils.data.sampler import SubsetRandomSampler
+
+print(os.path.dirname(os.path.dirname(__file__)))
+CONFIG_PATH = os.path.join(
+    os.path.dirname(os.path.dirname(__file__)), "metadata", "config_result_paper.json",
+)
+
+
+def test_trainer() -> None:
+    """function showing how to use the train function
+    """
+    sys.path.append(".")
+    os.chdir(os.path.dirname(os.path.dirname(__file__)))
+
+    from pytorch import train
+
+    n_features = 10
+    n_samples = 1000
+    X = torch_randint(
+        10, size=(n_samples, n_features), dtype=torch_float
+    )  # (n_samples, n_features)
+    Y = torch_randint(3, size=(n_samples, 1), dtype=torch_float)  # (n_samples,)
+
+    dataset = TensorDataset(X, Y)
+
+    with open(CONFIG_PATH) as json_file:
+        train_args = json.load(json_file)
+
+    train_dataloader, val_dataloader, _ = train_val_test_dataloaders(
+        dataset, batch_size=train_args.pop("batch_size")
+    )
+    train_args.update(
+        {
+            "train_dataloader": train_dataloader,
+            "val_dataloader": val_dataloader,
+            "n_features": n_features,
+        }
+    )
+    logging.info("The train function is tested with the following parameters:")
+    logging.info(pformat(train_args))
+    model = train(**train_args)
+
+    assert isinstance(model, nn.Module)
+
+
+def train_val_test_dataloaders(
+    dataset: object, batch_size: int, val_size: float = 0.2, test_size: float = 0.2, seed: int = 1,
```

Comment:
val_size -> val_frac ?
test_size -> test_frac ?

Reply:
It's changed!